charts/partials/templates/_tolerations.tpl

```diff
@@ -0,0 +1,4 @@
+{{- define "linkerd.tolerations" -}}
+tolerations:
+{{ toYaml .Values.tolerations | trim | indent 2 }}
+{{- end -}}
```

Comment:
nit: Can you add a extra line to this file?

Reply:
updated in [2611c65](https://github.com/linkerd/linkerd2/pull/4626/commits/2611c65776aa8063bb87d73054ca6356574126e6)